.teamcity/run_ci.sh

```diff
@@ -229,6 +229,12 @@ function mount_other_volumes() {
     reuse_or_start_volume "${faster_rcnn_volume}" "${faster_rcnn_image}"
     volume_conf="${volume_conf} --volumes-from ${faster_rcnn_volume}"
 
+    # SMOKE
+    local smoke_volume="apollo_smoke_volume_${USER}"
+    local smoke_image="${DOCKER_REPO}:smoke_volume-yolo_obstacle_detection_model-${TARGET_ARCH}-latest"
+    reuse_or_start_volume "${smoke_volume}" "${smoke_image}"
+    volume_conf="${volume_conf} --volumes-from ${smoke_volume}"
+
```

Comment:
Since only x86_64 volume was ready, we'd better add code like
```
if [[ "$(uname -m)" == "x86_64" ]]; then
    your logic here
fi

Reply:
May you help build an arm volume?